index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="./css/style.css">
+  <title>Tribute Project</title>
+</head>
+
+<body>
+  <div id="main">
+    <div class="title text-align-center">
+      <h1>Dr. Norman Borlaug</h1>
+    </div>
```

Comment:
- [ ] Optional
     I'd suggest wrapping this `h1` in a `header` tag instead of a `div` as it makes it much more semantic.

Reply:
Here, you could have explained why using semantic tags is of value to the author, does it affect any metrics or/and improve user experience?